debian/control

```diff
@@ -795,6 +825,21 @@ Description: Python 2 libraries for the Ceph librbd library
  This package contains Python 2 libraries for interacting with Ceph's
  RBD block device library.
 
+Package: python-rbd-dbg
+Architecture: linux-any
+Section: debug
```

Comment:
should have 
```
Priority: extra
Depends: python-dbg //// python3-dbg for python3 package
```

see https://wiki.debian.org/Python/DbgBuilds and https://wiki.ubuntu.com/PyDbgBuilds

Reply:
fixed.  also fixed some of the >= versions to = versions.